zenoh/src/lib.rs

```diff
@@ -364,26 +355,58 @@ pub mod config {
     pub use zenoh_config::*;
 }
 
-#[doc(hidden)]
-#[cfg(all(feature = "unstable", feature = "plugins"))]
-pub mod plugins {
-    pub use crate::api::plugins::{
-        PluginsManager, Response, RunningPlugin, RunningPluginTrait, ZenohPlugin, PLUGIN_PREFIX,
-    };
-}
+#[cfg(all(feature = "internal", not(feature = "unstable")))]
```

Comment:
If you want to do something like that, you should add `"unstable"` as a dependency of the `"internal"` feature.
But I don't see why you should do that. *internal* and *unstable* are orthogonal concepts. I think every plugins/bindings should be able to compile with *internal* only.

Reply:
I disagree, everything *internal* should not be treated by definition, as it is `doc(hidden)`. User should not care, because user should not use. And the user who cares because writing plugins/bindings already knows that *internal* API is *unstable*, because he cares.

Reply:
I prefer to classify it from user point of view like this:
- `unstable` - normal end user API which just not stabilized yet
- `internal` - end user API which exposes internal implementation details which are necessary for performing some tasks. This is unstable by definition and forever, the internal implementation is that we are free to change. But this doesn't mean that it's always will be `#[doc(hidden)]`, maybe it will make sense to document it, for plugins for example.

So I want to keep this scheme. If you want to make separation between `internal` and `unstable`, let's do it in separate update. This update just formalizes existing practice: now practically all internal api is marked unstable

Reply:
Ok.